launcher/src/test/java/org/apache/spark/launcher/SparkSubmitCommandBuilderSuite.java

```diff
@@ -344,8 +365,15 @@ private SparkSubmitCommandBuilder newCommandBuilder(List<String> args) {
     return newCommandBuilder(args).buildCommand(env);
   }
 
-  private void testCLIOpts(String opt) throws Exception {
-    List<String> helpArgs = Arrays.asList(opt, "driver-20160531171222-0000");
+  private void testCLIOpts(String appResource, String opt, List<String> params) throws Exception {
+    List<String> helpArgs = new ArrayList<>();
```

Comment:
Since you're changing this, probably better to rename the variable since these are not help arguments.

Reply:
Renamed.